explainaboard/metrics/nlg_meta_evaluation.py

```diff
@@ -18,6 +19,142 @@
 from explainaboard.utils.typing_utils import narrow, unwrap_or
 
 
+@dataclass
+@common_registry.register("CorrelationNLGConfig")
+class CorrelationNLGConfig(MetricConfig):
+    """Configuration of a correlation for general NLG tasks.
+
+    Args:
+        level: there are following levels: sample level, system level and dataset level
+                See more details: https://aclanthology.org/2020.emnlp-main.751.pdf
+        func_name: the method for calculating the correlation (e.g., Spearman)
```

Comment:
- "func_name" does not tell what this value is. Some other names such as "correlation" is suitable.
- You also need to add the full list of applicable values for this member. The example "Spearman" is not feasible.

Reply:
sounds good